src/jquery.uls.core.js

```diff
@@ -189,6 +189,10 @@
 			}
 
 			this.visible();
+
+			// Hide the selector panel when clicking outside of it
+			hider = $.proxy( this.cancel, this );
+			setTimeout( function() { $( 'html' ).one( 'click', hider ); } );
```

Comment:
Probably no measurable difference. This only sets the event listener when needed, for the other one it is always there.